opamp/http_client.py

```diff
@@ -183,7 +183,7 @@ def get_agent_description(self) -> opamp_pb2.AgentDescription: # type: ignore
                 value=anyvalue_pb2.AnyValue(string_value=self.instance_uid)
             ),
             anyvalue_pb2.KeyValue(
-                key=ResourceAttributes.PROCESS_PID,
+                key="process.vpid", 
```

Comment:
consider using `PROCESS_VPID` constant which is used for the detector instead of explicitly writing the string here